featuretools_tsfresh_primitives/tests/test_primitives.py

```diff
@@ -2,76 +2,77 @@
 import os
 
 from pytest import fixture
-import featuretools as ft
 
+import featuretools as ft
```

Comment:
Isort thinks featuretools is part of this library, we should add an isort section to setup.cfg and add `known_third_party=featuretools` so it knows to consider it a third party library

Reply:
I see, thanks. I update the config.